File: TTs/TT03/C-Cpp-Java/IfElse.c
#include <stdio.h>

int main (void) {
	int num;
	//
	printf("Digite um numero: ");
	scanf("%d", &num);
	//
	if(num < 0)
		printf("O numero %d eh negativo.\n", num);
	else	
		if (num == 0)
			printf("O numero %d eh nulo/neutro.\n", num);
		else
			printf("O numero %d eh positivo.\n", num);
	return 0;

}

File: TTs/TT03/C-Cpp-Java/IfElse.cs
using System;

class MainClass {
	public static void Main (string[] args) {
		int num;
		//
		Console.Write("Digite um numero: ");
		num = int.Parse(Console.ReadLine());
		//
		if(num < 0)
			Console.WriteLine("O numero "+num+" eh negativo.");
		else
			if (num == 0)
				Console.WriteLine("O numero "+num+" eh zero/nulo.");
			else
				Console.WriteLine("O numero "+num+" eh positivo.");
	}
}
